README.md

```diff
@@ -3,41 +3,41 @@
 Performance measurement app for KUKSA databroker.
 
 ```
-[00:00:00]  [========================================================================================================]    1000/1000    iterations
+[00:00:09] Group: Frame 1 | Cycle(ms): 10 | Current latency: 0.868 ms [===========================================================================================================]    1000/1000    iterationsSummary:
 
-Summary:
-  API: SdvDatabrokerV1
-  Elapsed time: 0.16 s
-  Rate limit: None
+Group: Frame 1 | Cycle time(ms): 10
+  API: KuksaValV2
+  Elapsed time: 9.99 s
+  Rate limit: 10 ms between iterations
   Sent: 1000 iterations * 1 signals = 1000 updates
```

Comment:
--help shows that "iterations" have been removed and replaced by "seconds"?

Reply:
Yes it does.

```
./target/release/databroker-perf --help
Usage: databroker-perf [OPTIONS]

Options:
  -r, --run-seconds <RUN_SECONDS>   Number of seconds to run [default: 8]
      --api <API>                   Api of databroker [default: kuksa.val.v1] [possible values: kuksa.val.v1, kuksa.val.v2, sdv.databroker.v1]
      --host <HOST>                 Host address of databroker [default: http://127.0.0.1]
      --port <PORT>                 Port of databroker [default: 55555]
      --skip-seconds <RUN_SECONDS>  Seconds to run (skip) before measuring the latency [default: 4]
      --detailed-output             Print more details in the summary result
      --test-data-file <FILE>       Path to test data file
      --run-forever                 Run the measurements forever (until receiving a shutdown signal)
  -v, --verbosity <LEVEL>           Verbosity level. Can be one of ERROR, WARN, INFO, DEBUG, TRACE [default: WARN]
  -h, --help                        Print help
  -V, --version                     Print version

```